pulsar-client-cpp/python/functions/context.py

```diff
@@ -50,9 +50,18 @@ def get_message_id(self):
     pass
 
   @abstractmethod
-  def get_topic_name(self):
+  def get_input_topic_name(self):
```

Comment:
Can we see if java also has the same interface?

Reply:
I updated the corresponding Java method as well. I think that `getTopicName`/`get_topic_name` is slightly confusing given that there are so many topics in play. Happy to revert if you disagree.

Reply:
Actually the method applies to the message that is currently being processed. So maybe we should update that to get_current_msg_topic_name or something equivalent?

Reply:
Good idea. I'll update.